packages/tablev2/tablev2.test.tsx

```diff
@@ -260,7 +259,9 @@ describe("Table v2", () => {
       );
 
       expect(sorterFn).toHaveBeenCalledWith("asc", 1);
-      component.find("#name button").simulate("click");
+
+      fireEvent.click(container.querySelector("#name button")!);
```

Comment:
@manav-06 this is because with querySelector operator it is not always certain we will get that element. I would recommend using getByTestId and assigning the proper test id to the element.

Reply:
@taksuparth @nataliepina please review.